ansible/deploy.py

```diff
@@ -0,0 +1,162 @@
+import click
```

Comment:
I suggest making this file executable and adding a shebang so you can invoke it as `./deploy.py`:

```suggestion
#!/usr/bin/env python
import click
```